src/taglib.pyx

```diff
@@ -13,13 +13,10 @@ from libcpp.utility cimport pair
 cimport ctypes
 
 version = '1.3.0'
-
-cdef void init():
+def disableDebug():
     cdef ctypes.PytaglibDebugListener *_dl = new ctypes.PytaglibDebugListener()
```

Comment:
Minor comment: This introduces a memory leak when `disableDebug` is called repeatedly.